command runs "per-project" (#42)
ddev/ddev-browsersync: README.md

- [Introduction](#introduction)
- [Getting Started](#getting-started)

>   - [Note](#note)

Reviewer: I would either omit this since “Note” takes space and attention without conveying anything—like a road sign that says “sign ahead”—or relabel it to something like “v2.5.0 Changes”.

I’m suggesting omission because it feels like change notes should be part of the changelog or release notes, and because the user is prompted to delete the orphaned global file and there’s presumably no other harm to leaving it around.

```suggestion
```

EG.
"External: <http://d9.ddev.site:3000>" => Access on **<https://d9.ddev.site:3000>**

> ### Note

Reviewer: From above, I would omit or relabel this.

```suggestion
```

> From version `2.5.0`, this addon uses a "per-project" approach. This will not affect usage as project-level command have higher priority than global commands.

Reviewer: Attempt to emphasize callout while being more concise about it:

```suggestion
> :bulb: This add-on moves to a per-project approach in v2.5.0+. You can safely delete the global `~/.ddev/commands/web/browsersync` once you’re on v2.5.0 or higher—this will not affect usage.
```

> When installing current versions, a warning is displayed if the global command (`~/.ddev/commands/web/browsersync`) is detected. You may safely delete his file.

Reviewer: Can omit this with above edit.

```suggestion
```

> If you run `ddev browsersync` from multiple local projects and receive a `Error: unknown command "browsersync" for "ddev"`, run the following command to add the command to the project as needed.

Reviewer: Reframe for the context of a singular project, since it sounds like the person would need to run the command per-project anyway. (Would someone likely be running from multiple projects simultaneously?) Use a colon to pitch to the command we explicitly mention.

```suggestion
If you run `ddev browsersync` from a local project and get `Error: unknown command "browsersync" for "ddev"`, run the following to add the command to the project:
```
